tutor/routes/home.py

```diff
@@ -12,6 +12,7 @@ def index_route():
 def register_route():
     return register.register()
 
+
```

Comment:
it doesn't seem like this empty line really needs to be added or belongs in this PR.